src/Controls/src/Core/HandlerImpl/SwipeView.Impl.cs

```diff
@@ -123,6 +134,75 @@ void UpdateMargin()
 				swipeView.RequestClose(new SwipeViewCloseRequest(false));
 		}
 
+		void UpdateLogicalChildren()
+		{
+			if (!_isOpen)
+			{
+				ClearLogicalChildren();
+				return;
+			}
+
+			var swipeItems = GetSwipeItemsByDirection(_swipeDirection);
+
+			if (swipeItems is null)
+				return;
+
+			foreach (var swipeItem in swipeItems)
+				AddLogicalChild((Element)swipeItem);
+		}
+
+		SwipeItems? GetSwipeItemsByDirection(SwipeDirection? swipeDirection)
+		{
+			SwipeItems? swipeItems = null;
+
+			switch (swipeDirection)
+			{
+				case SwipeDirection.Left:
+					swipeItems = RightItems;
+					break;
+				case SwipeDirection.Right:
+					swipeItems = LeftItems;
+					break;
+				case SwipeDirection.Up:
+					swipeItems = BottomItems;
+					break;
+				case SwipeDirection.Down:
+					swipeItems = TopItems;
+					break;
+			}
+
+			return swipeItems;
+		}
+
+		internal void AddLogicalChild(Element element)
+			=> AddLogicalChildInternal(element);
+
+		internal new void ClearLogicalChildren()
```

Comment:
I think we can delete this. The `ClearLogicalChildren` on the base class should handle the clear correctly on the overriden backing store.

Reply:
Right, updated.